From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Jordan <immoloism@gmail.com>
Date: Wed, 7 Dec 2022 13:19:52 +0000
Subject: [PATCH] Fix build with clang16

Add missing include directives for make_dashed, set_cursor_busy.

Ref: https://bugs.gentoo.org/875158
Origin: Gentoo
Upstream-Status: Unknown
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 src/xo-image.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/src/xo-image.c
+++ b/src/xo-image.c
@@ -25,6 +25,8 @@
 #include "xo-support.h"
 #include "xo-image.h"
 #include "xo-misc.h"
+#include "xo-selection.h"
+#include "xo-paint.h"
 
 // create pixbuf from buffer, or return NULL on failure
 GdkPixbuf *pixbuf_from_buffer(const gchar *buf, gsize buflen)
